components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml

```diff
@@ -33,7 +33,6 @@ spec:
         - quality-dashboard
         - spi-system
         - group-sync-operator
-        - tekton-chains
```

Comment:
Just realized @ramessesii2 @Roming22 - this will update production as well, since it is the base overlay, correct?

I think we want to isolate the removal solely to staging, no?

Reply:
I didn't bother to isolate it because `tekton-chains` is already deployed via `openshift-pipelines` operator in the production as well and so I though it wouldn't make a difference in the production.
Anyways, to be on the safe side, I'll do a separate PR to address it. Thanks @gabemontero

Reply:
Or maybe you'd like to see how this works out in staging? So, I'll do some patch so that it's only reflected in staging.

Reply:
Update: https://github.com/redhat-appstudio/infra-deployments/pull/2364/files#diff-db4d5ee061a063bfe1c269a1e0c5709c900bb3e199bd1f24bb3e7c4a87d8b955R6